apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -148,6 +148,19 @@ statements(pgsql) ->
       <<"UPDATE groups SET last_updated_by= $1, updated_at= $2, name= $3"
         "WHERE id= $4">>},
      {delete_group_by_id, <<"DELETE FROM groups WHERE id= $1">>},
+     {list_policies_for_org, <<"SELECT name, policy_group FROM policies WHERE org_id= $1">>},
```

Comment:
That sounds good, as long as we have migration cases in mind if real users will have data in these schemas while we develop them.